docs/continuous_integration.md

```diff
@@ -1,16 +1,34 @@
 # vHive CI guide
 
 ## Deploying GitHub self-hosted runners
-* Create Create GitHub personal access token for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
+
+### Deploying integration tests workflow runners
+1. Create GitHub personal access token for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
+
+2. Clone the repository.
+
+```bash
+git clone https://github.com/vhive-serverless/vhive && cd vhive
+```
+
+3. Setup the desired number of runners:
+
+```bash
+./scripts/github_runner/setup_integ_runners.sh <num of runners> <GitHub Account> <Github PAT> [restart]
+```
+
+### Deploying CRI tests workflow runners
+
+1. Clone the repository.
+
+```bash
+git clone https://github.com/vhive-serverless/vhive && cd vhive
+```
+
+2. Setup the runner for the desired sandbox:
+
 ```bash
-# clone your repo
-git clone <repo URL> && cd vhive
-# setup host 
-./scripts/github_runner/setup_runner_host.sh
-# start integration test runners
-./scripts/github_runner/start_runners.sh <num of runners> https://github.com/<OWNER>/<REPO> <Github Access key> integ
-# start cri test runners
-./scripts/github_runner/start_runners.sh <num of runners> https://github.com/<OWNER>/<REPO> <Github Access key> cri
+./scripts/github_runner/setup_bare_metal_runner.sh <GitHub organization> <Github PAT> <runner label> <runner name> <sandbox>
```

Comment:
Please list the supported labels and sanboxes

Reply:
Removed label parameter, added a list of sandboxes.

Reply:
where? did you push?

Reply:
I pushed the wrong branch, should be up to date now.